application-packages/DeployProcess_Stacker.json

```diff
@@ -0,0 +1,98 @@
+{
+    "processOffering": {
+        "process": {
+            "id": "Stacker",
+            "title": "Stacker",
+            "abstract": "",
+            "keywords": [],
+            "owsContext": {
+                "offering": {
+                    "code": "http://www.opengis.net/eoc/applicationContext/cwl",
+                    "content": {
+                        "href": "https://raw.githubusercontent.com/crim-ca/testbed14/master/application-packages/StackCreation-graph-json-remote.cwl"
+                    }
+                }
+            },
+            "inputs": [
+                {
+                    "id": "files",
+                    "title": "Input Image",
+                    "formats": [
+                        {
+                            "mimeType": "application/zip",
+                            "default": true
+                        }
+                    ],
+                    "minOccurs": 1,
+                    "maxOccurs": "unbounded",
+                    "additionalParameters": [
```

Comment:
No idea what this refers to...

Reply:
unbounded is the official term used for infinite

Reply:
Was referring to the additionalParameters {}, what do these values refer too?

Reply:
Alice should indicate the input(s) that needs to be replaced by OpenSearch own parameters.

Reply:
So this input is expected as is by the ADES but the EMS describe process will not show this input but rather the aoi, toi, etc.